health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -1530,4 +1530,108 @@ export const UICustomizations = {
+    
+      // --- Handle serviceCode ---
+      let serviceCodes = _.clone(data.body.inbox.moduleSearchCriteria.serviceCode || []);
+      serviceCodes = serviceCodes?.map((row) => row?.serviceCode);
+      delete data.body.inbox.moduleSearchCriteria.serviceCode;
+      if (serviceCodes.length > 0) {
+        data.body.inbox.moduleSearchCriteria.complaintType = serviceCodes;
+      } else {
+        delete data.body.inbox.moduleSearchCriteria.complaintType;
+      }
+
+      delete data.body.inbox.moduleSearchCriteria.locality;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)